lib/absinthe/schema/notation.ex

```diff
@@ -1991,7 +1991,6 @@ defmodule Absinthe.Schema.Notation do
   defp default_name(Schema.DirectiveDefinition, identifier) do
     identifier
     |> Atom.to_string()
-    |> Absinthe.Utils.camelize(lower: true)
```

Comment:
This did succeed in `camelCasing` the directive in the test below, but it broke a bunch of other things.